src/components/screens/error-page.tsx

```diff
@@ -0,0 +1,25 @@
+import { type ReactElement } from "react";
```

Comment:
As in previous comments I believe this import can be removed same goes for the use of it as the return type and the comment referencing it in the JSDoc- I won't comment on these any more...

Reply:
removed